mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -1821,15 +1821,59 @@ func TestAccComputeInstance_secondaryAliasIpRange(t *testing.T) {
 					testAccCheckComputeInstanceHasAliasIpRange(&instance, "inst-test-secondary", "172.16.0.0/24"),
 				),
 			},
-			computeInstanceImportStep("us-east1-d", instanceName, []string{}),
+			computeInstanceImportStep("us-east1-d", instanceName, []string{"network_interface.0.alias_ip_range.0.ip_cidr_range", "network_interface.0.alias_ip_range.0.subnetwork_range_name", "network_interface.0.alias_ip_range.1.ip_cidr_range", "network_interface.0.alias_ip_range.1.subnetwork_range_name"}),
 			{
 				Config: testAccComputeInstance_secondaryAliasIpRangeUpdate(networkName, subnetName, instanceName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckComputeInstanceExists(t, "google_compute_instance.foobar", &instance),
 					testAccCheckComputeInstanceHasAliasIpRange(&instance, "", "10.0.1.0/24"),
 				),
 			},
+			computeInstanceImportStep("us-east1-d", instanceName, []string{"network_interface.0.alias_ip_range.0.ip_cidr_range", "network_interface.0.alias_ip_range.0.subnetwork_range_name", "network_interface.0.alias_ip_range.1.ip_cidr_range", "network_interface.0.alias_ip_range.1.subnetwork_range_name"}),
+		},
+	})
+}
+
+func TestAccComputeInstance_aliasIpRangeCommonAddresses(t *testing.T) {
```

Comment:
Will this actually be able to verify that the mechanism used to make the change is correct? If so, how?

Reply:
These test cases are not verifying this mechanism per se, but rather the end result. See your point though. 

If there is any way to catch the first PATCH call to the API on the fly in the test case I'd be happy to code that in. Just didn't find anything similar

Maybe a unit test for the checkForCommonAliasIp() function will be sufficient to cover this

Reply:
hmm... There's no (reasonable) way to catch the first PATCH call on the fly as far as I know, so it might just require verification of the calls in the debug logs.

but adding unit tests for checkForCommonAliasIp certainly wouldn't hurt!

Reply:
Added the unit tests but haven't found a way to get the provider debug log from within an acc test other than using `stdout`. Which doesn't seem to be safe considering `t.Parallel()`. 

Are there any similar implementations? 
And also do the tests in GitHub Actions run with `TF_LOG=DEBUG` or `TRACE`? To catch any logs to begin with.

Reply:
unit tests don't use the Terraform binary so `TF_LOG` wouldn't have any impact. using the `t.Errorf` and `t.Logf` should let you output anything that shouldn't halt the test.

Reply:
sorry, you said the acc test. Just a sec.

Reply:
yeah getting things to print inside acc tests can be tricky. I assume you're talking about logging resource behavior specifically? I think using the `"github.com/hashicorp/terraform-plugin-log/tflog"` package might be the canonical best practice (though we only use it in a few places - I think most resource use `log`, but I'm pretty sure that's just for historical reasons.

(Side note: Acc tests don't get run in GitHub Actions.)